src/cdk/stepper/stepper.ts

```diff
@@ -212,9 +213,12 @@ export class CdkStepper implements AfterViewInit, OnDestroy {
   ngAfterViewInit() {
     this._keyManager = new FocusKeyManager(this._stepHeader)
       .withWrap()
-      .withHorizontalOrientation(this._layoutDirection())
       .withVerticalOrientation(this._orientation === 'vertical');
 
+    (this._dir ? this._dir.change.asObservable() : obaservableOf())
```

Comment:
@crisbeto can you remove the `asObservable` here? There are places in Google where people provide a fake `Directionality` with just an `Observable` as `change` instead of a `Subject`

(the real API should really be an `Observable` as well)